### PR TITLE
[4.0] com_finder remove text-center

### PR DESCRIPTION
**Pull request by brianteeman**
Removes text-center from index and maps view for some columns to match the style in the rest of the admin for columns of a similar type

### CODE REVIEW

File: administrator/components/com_finder/tmpl/index/default.php [hunk 1]
```diff
@@ -50,14 +50,14 @@
 							<th scope="col">
 								<?php echo HTMLHelper::_('searchtools.sort', 'JGLOBAL_TITLE', 'l.title', $listDirn, $listOrder); ?>
 							</th>
-							<th scope="col" style="width:10%" class="d-none d-md-table-cell text-center">
+							<th scope="col" style="width:10%" class="d-none d-md-table-cell">
 								<?php echo HTMLHelper::_('searchtools.sort', 'COM_FINDER_INDEX_HEADING_INDEX_TYPE', 't.title', $listDirn, $listOrder); ?>
 							</th>
 							<th scope="col" style="width:10%" class="d-none d-md-table-cell text-center">
```
Comment (Quy) [inline administrator/components/com_finder/tmpl/index/default.php:54]: Remove here too. Go to `Search Filters`, see `Created On` column where it is left-aligned.

Comment (brianteeman) [reply administrator/components/com_finder/tmpl/index/default.php:54]: the new template screws this up anyway

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on e91f0123ddb6018011eee6da5b5ca6993bd4caa4<br /><br />Seems to be ok.

But will the new backend template not change this all again anyway?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25560">issues.joomla.org/tracker/joomla-cms/25560</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on e91f0123ddb6018011eee6da5b5ca6993bd4caa4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25560">issues.joomla.org/tracker/joomla-cms/25560</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25560">issues.joomla.org/tracker/joomla-cms/25560</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
thanks

**Pull request merged**